### PR TITLE
Rating a search result blog from Karel

### PR DESCRIPTION
**Pull request by kmpungi**
A blog from Karel which explain how to rate a Search Result, Customizing
Search Results Ranking and Ranking Functions

### CODE REVIEW

**Comment by kmpungi** [comment]
A blog from Karel which explain how to rate a Search Result, Customizing
Search Results Ranking and Ranking Functions

File: _posts/2015-03-20-Rating-a-Search-Result-Customizing-Search-Results-Ranking-and-Ranking-Functions.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+## Customizing Search Results Ranking
+
+CES uses 18 ranking factors divided into 6 types to calculate the relevance score of documents. CES natively uses pre-tuned ranking weights that are likely to be satisfying most of the time. 
+Nevertheless, fine-tuning these factors can sometimes increase accuracy, especially for specific situations. 
+Click [here](http://onlinehelp.coveo.com/en/CES/7.0/Administrator/Customizing_Search_Results_Ranking.htm) for more information.
+
+## Ranking Functions
+
+Ranking functions are like [Query Functions](https://developers.coveo.com/display/SearchREST/Query+Function) ,but the result of the computation is used to influence ranking, just like Query Ranking Expressions.
```
Comment (pastjean) [inline _posts/2015-03-20-Rating-a-Search-Result-Customizing-Search-Results-Ranking-and-Ranking-Functions.md]: space comma nospace should be nospace comma space

File: _posts/2015-03-20-Rating-a-Search-Result-Customizing-Search-Results-Ranking-and-Ranking-Functions.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+## Customizing Search Results Ranking
+
+CES uses 18 ranking factors divided into 6 types to calculate the relevance score of documents. CES natively uses pre-tuned ranking weights that are likely to be satisfying most of the time. 
+Nevertheless, fine-tuning these factors can sometimes increase accuracy, especially for specific situations. 
+Click [here](http://onlinehelp.coveo.com/en/CES/7.0/Administrator/Customizing_Search_Results_Ranking.htm) for more information.
+
+## Ranking Functions
+
+Ranking functions are like [Query Functions](https://developers.coveo.com/display/SearchREST/Query+Function) ,but the result of the computation is used to influence ranking, just like Query Ranking Expressions. 
+The computation is done in the Ranking's first phase, meaning that all documents will have a chance to receive the Ranking Function's boosts.
```
Comment (pastjean) [inline _posts/2015-03-20-Rating-a-Search-Result-Customizing-Search-Results-Ranking-and-Ranking-Functions.md]: not sure about capital letter on "Rankin Function" and other "Ranking" should be emphasized or bold instead

File: _posts/2015-03-20-Rating-a-Search-Result-Customizing-Search-Results-Ranking-and-Ranking-Functions.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+![Modify group for Collaborative rating]({{ site.baseurl }}/images/ModifygroupforCollaborativerating.png)
+
+_Removing or modifying a personal appreciation_:
+
+To delete your personal appreciation for a document, right-click on the yellow stars:
+The document rank is immediately adjusted according to the automatic relevance score in the search results list and the appropriate number of gray stars![Two grey stars score]({{ site.baseurl }}/images/Twogreystarsscore.png) appears.
+
+To modify your personal appreciation:
+Click the appropriate yellow star to assign your personal appreciation. 
+For example, if the personal appreciation was three yellow stars![Full Yellow Stars]({{ site.baseurl }}/images/FullYellowStars.png),click on the first yellow star and your personal appreciation will be one![One Yellow Stars]({{ site.baseurl }}/images/OneYellowStars.png)  star instead of three.
```
Comment (pastjean) [inline _posts/2015-03-20-Rating-a-Search-Result-Customizing-Search-Results-Ranking-and-Ranking-Functions.md]: Pictures are not same size but represent same number of stars, should be normalized

File: _posts/2015-03-20-Rating-a-Search-Result-Customizing-Search-Results-Ranking-and-Ranking-Functions.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+So when I perform a “tax” query, this document will be ranked higher on my search result page compared to the search result page of the other user from my group. 
+The reason behind that is that personal appreciation prevails on collaborative rating meaning that, once you rate a search result, its collaborative rating score is no longer taken into account. 
+
+Now If I asked to someone from the same group as mine to check the collaborative rating of the same document, it will be higher than what it was before I gave it a personal appreciation.
+
+![Collaborative rating 2 grey star]({{ site.baseurl }}/images/Collaborativerating2greystar.png)
+
+Note: You need to wait until the next update of document rating in order for your rating to take effect.
+If you go to CES Admin tool – Configuration – Schedules – System
```
Comment (pastjean) [inline _posts/2015-03-20-Rating-a-Search-Result-Customizing-Search-Results-Ranking-and-Ranking-Functions.md]: `CES Admin tool – Configuration – Schedules – System` escape this in backticks

File: _posts/2015-03-20-Rating-a-Search-Result-Customizing-Search-Results-Ranking-and-Ranking-Functions.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+Check the schedule of “Update Document Rating”, by default it’s once a week, but you can change it as you like.
+
+![Update Document Rating]({{ site.baseurl }}/images/UpdateDocumentRating.png)
+
+_Where should I go to create/Modify group for Collaborative rating_:
+
+You can do it from in the CES Admin tool – Index – Ranking – Collaborative Rating.
```
Comment (pastjean) [inline _posts/2015-03-20-Rating-a-Search-Result-Customizing-Search-Results-Ranking-and-Ranking-Functions.md]: same escape in backtick

File: _posts/2015-03-20-Rating-a-Search-Result-Customizing-Search-Results-Ranking-and-Ranking-Functions.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+author:
+  name: Karel Mpungi
+  bio: Product Expert, Advanced Enterprise Search Platform
+  image: kmpungi.jpg
+---
+
+
+## Rating a Search Result:
+
+_What is the advantage of rating a search result:_
```
Comment (pastjean) [inline _posts/2015-03-20-Rating-a-Search-Result-Customizing-Search-Results-Ranking-and-Ranking-Functions.md]: 3 sharps makes another title

File: _posts/2015-03-20-Rating-a-Search-Result-Customizing-Search-Results-Ranking-and-Ranking-Functions.md [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+
+Rating a search result is useful when you find that its automatic ranking in the current query is not appropriate from your point of view or that of your group. 
+This immediately changes its position in the search results page. You or other members of your group also benefit from this corrected rating the next time the same query is performed.
+
+It’s important that you understand the difference between “Personal appreciation” and “Collaborative rating”
+
+_Personal appreciation:_
```
Comment (pastjean) [inline _posts/2015-03-20-Rating-a-Search-Result-Customizing-Search-Results-Ranking-and-Ranking-Functions.md]: 3 sharps

File: _posts/2015-03-20-Rating-a-Search-Result-Customizing-Search-Results-Ranking-and-Ranking-Functions.md [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+
+Personal appreciation is a score that you, as a user, can assign to a document or item to reflect your perception of its relevance to the query. By rating a search result on a 1 to 3 yellow stars scale![Full Yellow Stars]({{ site.baseurl }}/images/FullYellowStars.png)
+you can influence the search result ranking. The Coveo Platform uses your personal appreciation to immediately modify the ranking of a search result and to calculate its collaborative rating.
+
+
+_Collaborative rating:_
```
Comment (pastjean) [inline _posts/2015-03-20-Rating-a-Search-Result-Customizing-Search-Results-Ranking-and-Ranking-Functions.md]: 3 sharps ###

**Comment by kmpungi** [comment]
The change has been made, so the missing picture should be displayed

File: _posts/2015-03-20-Rating-a-Search-Result-Customizing-Search-Results-Ranking-and-Ranking-Functions.md [hunk 9]
```diff
@@ -0,0 +1,86 @@
...
+Click [here](http://onlinehelp.coveo.com/en/CES/7.0/Administrator/Customizing_Search_Results_Ranking.htm) for more information.
+
+## Ranking Functions
+
+Ranking functions are like [Query Functions](https://developers.coveo.com/display/SearchREST/Query+Function) ,but the result of the computation is used to influence ranking, just like Query Ranking Expressions. 
+The computation is done in the Ranking's first phase, meaning that all documents will have a chance to receive the Ranking Function's boosts.
+If NormalizeWeight is enabled, the entire value domain of the function will be taken into account to give a normalized boost ranking from 0 to 600. 
+By enabling ranking information, it is possible to see the actual boost value provided by the functions, for each document.
+
+Click [here](https://developers.coveo.com/display/SearchREST/Ranking+Function) for more information.
```
Comment (pastjean) [inline _posts/2015-03-20-Rating-a-Search-Result-Customizing-Search-Results-Ranking-and-Ranking-Functions.md]: -> should be

    For more information about ranking functions look at the [Ranking function documentation](https://developers.coveo.com/display/SearchREST/Ranking+Function)

File: _posts/2015-03-20-Rating-a-Search-Result-Customizing-Search-Results-Ranking-and-Ranking-Functions.md (hunk 6 above)
Comment (kmpungi) [inline _posts/2015-03-20-Rating-a-Search-Result-Customizing-Search-Results-Ranking-and-Ranking-Functions.md]: Why do you want another title level?

File: _posts/2015-03-20-Rating-a-Search-Result-Customizing-Search-Results-Ranking-and-Ranking-Functions.md (hunk 7 above)
Comment (kmpungi) [inline _posts/2015-03-20-Rating-a-Search-Result-Customizing-Search-Results-Ranking-and-Ranking-Functions.md]: Same here

File: _posts/2015-03-20-Rating-a-Search-Result-Customizing-Search-Results-Ranking-and-Ranking-Functions.md (hunk 8 above)
Comment (kmpungi) [inline _posts/2015-03-20-Rating-a-Search-Result-Customizing-Search-Results-Ranking-and-Ranking-Functions.md]: Same here

File: _posts/2015-03-20-Rating-a-Search-Result-Customizing-Search-Results-Ranking-and-Ranking-Functions.md (hunk 4 above)
Comment (kmpungi) [inline _posts/2015-03-20-Rating-a-Search-Result-Customizing-Search-Results-Ranking-and-Ranking-Functions.md]: Done

File: _posts/2015-03-20-Rating-a-Search-Result-Customizing-Search-Results-Ranking-and-Ranking-Functions.md (hunk 5 above)
Comment (kmpungi) [inline _posts/2015-03-20-Rating-a-Search-Result-Customizing-Search-Results-Ranking-and-Ranking-Functions.md]: Done

File: _posts/2015-03-20-Rating-a-Search-Result-Customizing-Search-Results-Ranking-and-Ranking-Functions.md (hunk 3 above)
Comment (kmpungi) [inline _posts/2015-03-20-Rating-a-Search-Result-Customizing-Search-Results-Ranking-and-Ranking-Functions.md]: What do you mean by should be normalized, can you be more specific please?

File: _posts/2015-03-20-Rating-a-Search-Result-Customizing-Search-Results-Ranking-and-Ranking-Functions.md (hunk 1 above)
Comment (kmpungi) [inline _posts/2015-03-20-Rating-a-Search-Result-Customizing-Search-Results-Ranking-and-Ranking-Functions.md]: Done

File: _posts/2015-03-20-Rating-a-Search-Result-Customizing-Search-Results-Ranking-and-Ranking-Functions.md (hunk 2 above)
Comment (kmpungi) [inline _posts/2015-03-20-Rating-a-Search-Result-Customizing-Search-Results-Ranking-and-Ranking-Functions.md]: no more capital "R" or "F", done.

File: _posts/2015-03-20-Rating-a-Search-Result-Customizing-Search-Results-Ranking-and-Ranking-Functions.md (hunk 9 above)
Comment (kmpungi) [inline _posts/2015-03-20-Rating-a-Search-Result-Customizing-Search-Results-Ranking-and-Ranking-Functions.md]: Done

**Comment by pastjean** [comment]
:+1: changes integrated in its own branch in current project. i'll merge 
https://github.com/Coveo/coveo.github.io/tree/kmpungiarticle

**Pull request closed without merging**